pkg/controller/import-controller.go

```diff
@@ -23,6 +23,8 @@ const (
 	AnnImportPod = "cdi.kubevirt.io/storage.import.importPodName"
 	//LabelImportPvc is a pod label used to find the import pod that was created by the relevant PVC
 	LabelImportPvc = "cdi.kubevirt.io/storage.import.importPvcName"
+	//AnnBlankImage provides a const for our PVC blankImage annotation
+	AnnBlankImage = "cdi.kubevirt.io/storage.import.blankImage"
```

Comment:
@zvikorn 
I believe we decided on giving everything a 'source', so a blank image becomes

cdi.kubevirt.io/storage.import.source: blank

Then the controller can decided what to do based on the value of the source. I am creating a card to do the same for the existing sources.

Reply:
oh, I see. I thought that we will change everything together at some point. Ok, I will change it. Thanks!

Reply:
@awels I take my words back :-) I think that we should have a PR first to change the existing annotation, and based on that I will update my bank image feature. I do not want to change anything which is not related to the feature itself like creating one common annotation.  This will be in the other PR. What do you think?

Reply:
I agree, I don't think it will be that much work to do an intermediate PR with making the change to include the 'source'.